index.html

```diff
@@ -235,23 +246,27 @@ <h4 id="about-heading">Let's Connect</h4>
                                 <img src="asset/chevron-down.png" alt="arrow icon">
                             </button>
                         </li>
-                        <li class="language-item">
-                            <div>
-                                <img src="asset/js-icon.png" alt="js icon">
-                                <p>Javascript</p>
-                            </div>
-                        </li>
-                        <li class="language-item">
-                            <div>
-                                <img src="asset/html-icon.png" alt="html icon">
-                                <p>HTML</p>
-                            </div>
-                        </li>
-                        <li class="language-item">
-                            <div>
-                                <img src="asset/css-icon.png" alt="css icon">
-                                <p>CSS</p>
-                            </div>
+                        <li>
+                            <ul id="language-list">
+                                <li class="language-item">
+                                    <div>
+                                        <img src="asset/js-icon.png" alt="js icon">
+                                        <p>Javascript</p>
+                                    </div>
+                                </li>
+                                <li class="language-item">
+                                    <div>
+                                        <img src="asset/html-icon.png" alt="html icon">
+                                        <p>HTML</p>
+                                    </div>
+                                </li>
+                                <li class="language-item">
+                                    <div>
+                                        <img src="asset/css-icon.png" alt="css icon">
+                                        <p>CSS</p>
+                                    </div>
+                                </li>
+                            </ul>
```

Comment:
The **About Myself and **Languages** section should be more or less on the same line as per the Figma template. However, in your case, the **Languages** heading has a bit more padding which is pushing it a bit more down from the top line. Kindly fix this issue.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/220936392-fe31018c-e2e4-4b73-a8f0-d840baa74abf.png) | ![image](https://user-images.githubusercontent.com/33268927/220936150-f3cf95d3-ae11-46a1-a8a1-ac39dd845e69.png) |